commit 12fdf6192a9469fc79d9ebcbd4badc96991420d4 (refs/changes/49/3149/4)
Author: Mike Zeller <mike.zeller@joyent.com>
Date:   2018-02-06T15:18:12-08:00 (1 year, 8 months ago)
    
    ZAPI-819 Destroying a VM should should not report a failure when there are no NICs to remove
    ZAPI-820 Cleanup error messages in network validation
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/common/validation.js b/lib/common/validation.js
index 547fd47..d6cb5f9 100644
--- a/lib/common/validation.js
+++ b/lib/common/validation.js
@@ -2321,7 +2321,7 @@ exports.validateNetworks = function (params) {
     }
 
     if (errs.length) {
-        throw new errors.ValidationFailedError('Invalid VM update parameters',
+        throw new errors.ValidationFailedError('Invalid network parameters',
             errs);
     }
 
@@ -2553,4 +2553,4 @@ function hasPolymorphicParamWithName(fieldName, params) {
 
     return false;
 }
-exports.hasPolymorphicParamWithName = hasPolymorphicParamWithName;
\ No newline at end of file
+exports.hasPolymorphicParamWithName = hasPolymorphicParamWithName;
diff --git a/lib/workflows/destroy.js b/lib/workflows/destroy.js
index 3b5d4f4..1047018 100644
--- a/lib/workflows/destroy.js
+++ b/lib/workflows/destroy.js
@@ -19,7 +19,7 @@ var restify = require('restify');
 var common = require('./job-common');
 var nfsVolumes = require('./nfs-volumes');
 
-var VERSION = '7.1.1';
+var VERSION = '7.1.2';
 
 
 /*
@@ -112,7 +112,7 @@ function removeNics(job, cb) {
      */
 
     if (!macs || !Array.isArray(macs) || macs.length === 0) {
-        cb('No MAC addresses to remove');
+        cb(null, 'No MAC addresses to remove');
         return;
     }
 
